src/pygrambank/api.py

```diff
@@ -21,7 +21,7 @@ def sheets_dir(self):
 
     def iter_sheets(self):
         for p in sorted(self.sheets_dir.iterdir(), key=lambda i: i.stem):
-            if p.is_file() and p.name != '.gitattributes':
+            if p.is_file() and len(p.name) >= 4 and p.name[-4:] == '.tsv':
```

Comment:
If we can't get around macos recreating stuff, I'd opt for an explicit list of ignored names, i.e.
```python
if p.is_file() and p.name not in ['.gitattributes', '.DS_store']:
```

Reply:
yeah but we only know how to handle tsv, so we could just look for .tsv files? I've checked and gitignore blocks .DS_Store from being committed so it shouldn't show up in the repos.

Reply:
So my proposal would be a guard against other stuff making it into the repo by accident.

Reply:
The gitignore does block .DS_store, but unless I'm mistaken it'll still be in my folder on my machine, which is what I'm running check over. 

Can some other script somewhere check for which non tsv-files are in there whenever a new dump is made, that is different from this api.py, sheet.py and check structure?

Reply:
ok :thumb:

Reply:
:) I basically only need check for verifying commits and updating [the revisions list ](https://github.com/glottobank/Grambank/blob/master/For_coders/Grambank_languages_revision_completion.csv). For verifying commits, I agree it would be good if there was a check that non-tsv files weren't uploaded.

I don't know how to best do both of these things in one command.

Reply:
Maybe adding ".DS_Store" to the ignored filenames is alright for now? If any other files turn up, we can have a discussion then?